src/engine/SCons/Action.py

```diff
@@ -821,7 +816,14 @@ def read(self): return ''
                 def readline(self): return ''
                 def __iter__(self): return iter(())
             stdout = stderr = f()
-        return dummyPopen(e)
+        pobj = dummyPopen(e)
+    finally:
+        # clean up open file handles stored in parent's kw
+        import io
```

Comment:
Why import io?

Reply:
good catch. was needed for the py3 specific issubclass check, but I thought a less specific check might be more useful and the import got left.